content/en/docs/concepts/services-networking/service.md

```diff
@@ -548,6 +548,9 @@ status:
 
 Traffic from the external load balancer is directed at the backend Pods. The cloud provider decides how it is load balanced.
 
+For LoadBalancer type of Services, when there are more than one port defined, all
```

Comment:
Actually another tiny nit
```suggestion
For LoadBalancer type of Services, when there is more than one port defined, all
```

(English is weird!)

Reply:
Got it. Thanks.